aiidalab_widgets_base/computational_resources.py

```diff
@@ -902,16 +936,15 @@ def on_setup_computer(self, _=None):
         )
         try:
             computer = computer_builder.new()
-            self._configure_computer(computer)
+            self._configure_computer(computer, self.transport.value)
+            computer.store()
         except (
             ComputerBuilder.ComputerValidationError,
             common.exceptions.ValidationError,
             RuntimeError,
         ) as err:
-            self.message = f"Failed to setup computer {type(err).__name__}: {err}"
+            self.message = f"Computer setup failed! {type(err).__name__}: {err}"
```

Comment:
Made this shorter, since long error messages do not fit the widget. We'll need a better solution for this.